integrationtests/test_encoderdecoder_eo_v2_300_model_factory.yaml

```diff
@@ -27,8 +27,6 @@ trainer:
         monitor: val/loss
         filename: best-state_dict-{epoch:02d}
         save_weights_only: True
-      
-  max_epochs: 2
```

Comment:
why has this been removed? it will just run longer, correct? or are you checking on the correct result after the run?

Reply:
It was a duplicate of line 10.